### PR TITLE
7alip | Week1

### PR DESCRIPTION
**Pull request by 7alip**

### CODE REVIEW

File: Week1/homework/src/actions/citiesOfCountry.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { printTitle } = require('../helpers');
+const { myQuery } = require('../sql/db');
+
+module.exports.citiesOfCountry = async country => {
+  const list = await myQuery(`SELECT cities.name AS 'City Name' FROM countries INNER JOIN cities ON cities.country_code = countries.code WHERE countries.name = "${country}"`);
```
Comment (gajduk) [inline Week1/homework/src/actions/citiesOfCountry.js]: using JOIN already 👍

File: Week1/homework/src/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Optional Test Functions:
+------------------------
+  1. cities netherland
+  2. population cities rotterdam
+  3. orderBy countries surface_area desc 10
+  4. orderBy cities population desc 10
+  5. total countries population
+`);
+
+commander.command('help').action(() => commander.help());
```
Comment (gajduk) [inline Week1/homework/src/index.js]: wow, this was not requested but great job 🥇

File: Week1/homework/src/sql/seed-tables.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    const cities = await getData('cities');
+    const cityValues = cities.map(c => Object.values(c));
+    const cityColumns = Object.keys(cities[0]).join(', ');
+
+    const countries = await getData('countries');
+    const countryValues = countries.map(c => Object.values(c));
+    const countryColumns = Object.keys(countries[0]).join(', ');
+
+    // Insert multiple country
+    await myQuery(INSERT_TABLE('countries', countryColumns), countryValues);
```
Comment (gajduk) [inline Week1/homework/src/sql/seed-tables.js]: already using async await and promisify 👍

**Comment by 7alip** [comment]
Thank you so much for reviewing my code 🙂

**Pull request closed without merging**